### DISCUSSION TITLE
Marker not showing on React 18+ (Solved)

### DISCUSSION THREAD

**Discussion by TheRealFlyingCoder** [JustFly1984/react-google-maps-api#3230] [General]
Just dropping this solution here while the docs are waiting to be updated.

If there are issues with Markers not showing on your map and you are using React V18+ you just have to switch to the `MarkerF` variant of the component.

This might be the same with other elements but it's the only one i've had to touch for now :)

(Will delete this when the docs get updated with the Functional examples / content)

**Comment by kevinjavitz** [comment]
Thanks! I was stuck on this why my markers were not showing.

So is this project no longer maintained? What is the status? I know there is vis.gl/react-google-maps but it doesn't have components like DirectionsRenderer for route rendering

What is everybody doing?
